tools/check_tidy.py

```diff
@@ -213,7 +213,7 @@ def check_tidy(src_dir, options=None):
     allowed_files = ['CMakeLists.txt']
     clang_format_exts = ['.c', '.h']
     skip_dirs = ['deps', 'build', '.git', 'node_modules', 'coverage',
-                 'iotjs_modules']
+                 'iotjs_modules', 'config']
```

Comment:
IMHO we should not skip the whole config directory. It would be better to skip only the `config/tizen` directory